includes/admin.form.inc

```diff
@@ -5,23 +5,107 @@
  * Admin form functions for Islandora Altmetrics.
```

Comment:
Not exactly part of the PR, but would be appreciated if you could throw in a correction to this error in the documentation (should be Islandora Badges, no Islandora Altmetrics)